src/main/java/org/gelecekbilimde/scienceplatform/auth/model/request/RegisterRequest.java

```diff
@@ -20,30 +20,30 @@
 public class RegisterRequest {
 
 	@NotNull(message = "firstname cannot be null")
-	@Size(max = 25, message = "firstname should not be less than 25")
+	@Size(max = 25, min = 2 , message = "firstname should be more than 2 and less than 25")
```

Comment:
Message yazmamız şart mı? Spring'in exception'larını handle edip yazabiliriz. Ek olarak önce minimum değeri sonra maksimum değeri eklemeliyiz.
```suggestion
	@Size(min = 2, max = 25)
```